WORKSPACE

```diff
@@ -11,8 +11,8 @@ node_repositories(package_json = ["//:package.json"])
 
 git_repository(
     name = "build_bazel_rules_typescript",
-    remote = "https://github.com/bazelbuild/rules_typescript.git",
-    tag = "0.6.0",
+    remote = "https://github.com/gregmagolan/rules_typescript.git",
+    commit = "27d21f6953dbc61edae26c1da44dfcb08a7a325f",
```

Comment:
this is stale as I've rebased. new commit is 24f34f0659e08e82616d0a5d595f71faedc0e441. I tagged it as `angular-bazel-example-1` for this if you want to use the tag and I'll keep it current if rebasing.